### ISSUE TITLE
Remove package-lock.json from .gitignore

### ISSUE THREAD

**Issue by anshumanv** [fossasia/susi_desktop#73]
npm suggests that the package-lock.json file it meant to be committed to repositories.

https://docs.npmjs.com/files/package-lock.json

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
